is_core/generic_views/form_views.py

```diff
@@ -571,13 +574,39 @@ def get_success_url(self, obj):
         else:
             return self.request.get_full_path()
 
+    def _generate_export_types(self):
+        generated_export_types = []
+        for export_type in self._get_export_types():
+            generated_export_type = list(export_type)
+            try:
+                generated_export_types.append(
+                    list(export_type) + [get_converter(export_type[1]).media_type]
+                )
+            except KeyError:
+                raise ImproperlyConfigured('Export types improperly configured')
+        return generated_export_types
+
+    def _get_export_types(self):
+        return self.export_types if getattr(self, 'export_types', None) is not None else self.core.get_detail_export_types()
```

Comment:
Fungovalo by i tohle? Je to kratší
`return getattr(self, 'export_types', self.core.get_detail_export_types())`

Reply:
promiň kontroluješ špatný pull request :(